dynawo/3rdParty/libiidm/libiidm.cmake

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+cmake_minimum_required(VERSION 3.12)   
+
+include($ENV{DYNAWO_HOME}/dynawo/cmake/CPUCount.cmake)
+if(NOT DEFINED CPU_COUNT ) 
+  message(FATAL_ERROR "Le décompte des CPUs a échoué")
```

Comment:
No french.

Reply:
That little part is included in all others. It's a common part which can be mutualized. That code already exists in a few CMakeLists.txt but as copy/paste. I only take it apart to share it (which means testing is done good).

Reply:
done, nobody is perfect